console/src/range/ontology.tsx

```diff
@@ -181,7 +181,7 @@ const TreeContextMenu: Ontology.TreeContextMenu = (props) => {
         void Group.fromSelection(props);
         return;
       case "link": {
-        const toCopy = `synnax://cluster/${clusterKey}/range/${resources[0].id.key}`;
+        const toCopy = `synnax://cluster/${client.key}/range/${resources[0].id.key}`;
```

Comment:
This won't work. The client key is not equal to the cluster key. Use the cluster key instead.